src/components/home/elements/NavBar.vue

```diff
@@ -130,12 +135,111 @@ export default {
             }
         }
     },
+    data() {
+        return {
+            interval: 0,
+            response: '',
+            currentTime: '',
+            showHeight: false,
+            currentHeight: ''
+        }
+    },
+    created() {
+        this.showHeight = this.getHeightStatus()
+        if (this.showHeight) {
+            this.childByValue(this.showHeight)
+        }
+        this.currentHeight = this.getCurrentHeight()
+        this.currentTime = this.getCurrentTime()
+    },
     mounted() {
         jdenticon()
     },
     computed: {
     },
+    beforeDestroy() {
+        if (this.interval) {
+            clearInterval(this.interval)
+            this.interval = 0
+        }
+    },
     methods: {
+        getTime(date) {
+            const monthNames = ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN', 'JUL', 'AUG', 'SEP', 'OCT', 'NOV', 'DEC']
+            const d = new Date(date / 1e6)
+            let days = d.getDay()
+            let minutes = d.getMinutes()
+            let seconds = d.getSeconds()
+            if (minutes < 10) {
+                minutes = '0' + minutes
+            }
+            if (seconds < 10) {
+                seconds = '0' + seconds
+            }
+            if (days < 10) {
+                days = '0' + days
+            }
+            return d.getHours() + ':' + minutes + ':' + seconds + ',' + monthNames[d.getMonth()] + ' ' + days
+        },
+        getCurrentHeight() {
+            let oldHeight = 0
+            try {
+                oldHeight = window.localStorage.getItem('globalHeight')
+            } catch (e) {
+                oldHeight = 0
+            }
+            return oldHeight
+        },
+        getCurrentTime() {
+            let oldTime = 0
+            try {
+                oldTime = window.localStorage.getItem('time')
+            } catch (e) {
+                oldTime = 0
```

Comment:
the statement in catch, which the same as init statement, can be removed. (Also remove this case in Record.vue)